src/lib/gundb/UserStorageClass.js

```diff
@@ -626,16 +634,7 @@ export class UserStorage {
       trustPromise,
       AsyncStorage.getItem('GD_trust').then(_ => (this.trust = _ || {})),
       this.initFeed(),
-
-      // save ref to user
-      this.gun
-        .get('users')
-        .get(this.gunuser.is.pub)
-        .putAck(this.gunuser)
-        .catch(e => {
-          logger.error('save ref to user failed:', e.message, e)
-          throw e
-        }),
+      this.initProfile().catch(e => logger.error('failed initializing initProfile', e.message, e)),
```

Comment:
@serdiukov-o-nordwhale this is crashing in native creating all sorts of malformed calls from JS. if I comment this line and uncomment 599 it goes back to normal